_static/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="refresh" content="0; URL='/22.05/index.html'" />
```

Comment:
This looks fragile: what happens when we release a new stable version?
Is it possible to have an alias for the latest stable version?

Reply:
I couldn't find a way to render an HTML file with context from sphinx configuration.